frankenphp.c

```diff
@@ -832,7 +832,17 @@ static void *php_thread(void *arg) {
   cfg_get_string("filter.default", &default_filter);
   should_filter_var = default_filter != NULL;
 
-  while (go_handle_request(thread_index)) {
+  // perform work until go signals to stop
+  while (true) {
+    char *scriptName = go_frankenphp_before_script_execution(thread_index);
+
+    // if go signals to stop, break the loop
+    if (scriptName == NULL) {
+      break;
+    }
+
+    int exit_status = frankenphp_execute_script(scriptName);
+    go_frankenphp_after_script_execution(thread_index, exit_status);
```

Comment:
Nit:

```suggestion
  // loop until Go signals to stop
  char *scriptName = NULL;
  while (scriptName = go_frankenphp_before_script_execution(thread_index)) {
    go_frankenphp_after_script_execution(thread_index, frankenphp_execute_script(scriptName));
```

Reply:
That's actually a really nice refactor 👍